torch/library.h

```diff
@@ -85,6 +85,9 @@ class TORCH_API CppFunction final {
   // TODO: This is morally the same thing as KernelRegistrationConfig, but it's
   // opaque to the user.
 
+  // See Note [Plumbing Keys Through The Dispatcher]
+  enum class WithKeys { WITH_KEYS };
```

Comment:
Minor nit: although one is usually advised to `enum class` because it puts the enum entries in their own namespace, in this case it results in the wordy `CppFunction::WithKeys::WITH_KEYS` designation. An `enum` would work just fine and let you say `CppFunction::WITH_KEYS`, and is the current idiom in the codebase for using these.